crates/encoding/tests/net_serde.rs

```diff
@@ -20,8 +20,12 @@ struct SomeMessage {
     b: String,
     c: (bool, u64),
     d: Inner,
+    #[net_serde(skip)]
```

Comment:
This is usually to skip fields that uses foreign types that can't be annotated with `NetSerde`

Reply:
Actually no, this is only for the marker trait. With the skip, the `derive(NetSerde)` will not care of that field type implements `NetSerde` or not